test/test-morph-fill.jl

```diff
@@ -0,0 +1,21 @@
+using IceFloeTracker: morph_fill
+
+@testset "morp_fill.jl" begin
+    println("------------------------------------------------")
+    println("----------- Create Morp Fill Test --------------")
+
+    img = Bool[
+        0 0 0 0 0
+        0 1 1 1 0
+        0 1 0 1 0
+        0 1 1 1 0
+        0 0 0 0 0
+    ]
+
+    r = rand(1:100)
+
+    imgbig = repeat(img, r, r)
```

Comment:
It would repeat that basic block `r` times across and down. Say that `r` takes up the value 2. Then `imgbig` would look like this
```
[
    B B
    B B
]